gocognit118_test.go

```diff
@@ -0,0 +1,17 @@
+//go:build go1.18
+// +build go1.18
+
+package gocognit_test
+
+import (
+	"testing"
+
+	"github.com/uudashr/gocognit"
+	"golang.org/x/tools/go/analysis/analysistest"
+)
+
+func TestAnalyzerWithGenericMethods(t *testing.T) {
```

Comment:
What if we name it `TestAnalyzer_Generic`  or something not specific to methods?
The test that we have so far is to test the ability to show the struct/receiver name which has generics declaration. The method itself does not use the generic at all.

So in the future, we can start to add test/capability for gocognit to also works with generic methods and functions.
Ex:
```go
func SumNumbers[K comparable, V Number](m map[K]V) V {
    var s V
    for _, v := range m {
        s += v
    }
    return s
}
```

Reply:
changed the name to be plural - `TestAnalyzer_Generics`.